src/elements/toniq-slider/toniq-slider.story.tsx

```diff
@@ -0,0 +1,69 @@
+import {ComponentMeta} from '@storybook/react';
+import React from 'react';
+import {handleEventAsAction} from '../../storybook-helpers/actions';
+import {cssToReactStyleObject, toniqFontStyles} from '../../styles';
+import {ToniqSlider} from '../react-components';
+
+const componentStoryMeta: ComponentMeta<typeof ToniqSlider> = {
+    title: 'Elements/Toniq Slider',
+    component: ToniqSlider,
+};
+
+export default componentStoryMeta;
+
+export const mainStory = () => (
+    <article>
+        <h3
+            style={{
+                ...cssToReactStyleObject(toniqFontStyles.h3Font),
+            }}
+        >
+            Default
+        </h3>
+        <ToniqSlider onValueChange={handleEventAsAction} />
```

Comment:
Please add `onValueChange` to all examples in this story so actions fire for all of them.